docs/devenv_rhel8.md

```diff
@@ -103,9 +103,11 @@ sudo subscription-manager repos \
 sudo dnf localinstall -y ~/microshift/packaging/rpm/_rpmbuild/RPMS/*/*.rpm
 ```
 
-Download the OpenShift pull secret from the https://console.redhat.com/openshift/downloads#tool-pull-secret page. Copy it to `/etc/crio/openshift-pull-secret` and update its file permissions so CRI-O can use it when fetching container images:
+Download the OpenShift pull secret from the https://console.redhat.com/openshift/downloads#tool-pull-secret page. Copy it to `/etc/crio/openshift-pull-secret` and update its file permissions so `CRI-O` can use it when fetching container images. The `CRI-O` service should be restarted to pick up the change and enabled to run by default after reboot.
 ```bash
 sudo chmod 600 /etc/crio/openshift-pull-secret
+sudo systemctl restart crio
+sudo systemctl enable  crio
```

Comment:
Could we move this down into the section "Running MicroShift"? I'm hoping to keep (and even further increase) a clean separation between the first phase of building&installing MicroShift from RPMs (which should deliberately be identical to how users apply it) and the (optional) second phase for those developers that afterwards want to work with the MicroShift binary. Only the latter need to manually enable crio.